src/NServiceBus.Core/Transports/Msmq/QueuePermissions.cs

```diff
@@ -59,8 +59,6 @@ public static void SetPermissionsForQueue(MessageQueue queue, string account)
         {
 
             queue.SetPermissions(LocalAdministratorsGroupName, MessageQueueAccessRights.FullControl, AccessControlEntryType.Allow);
-            queue.SetPermissions(LocalEveryoneGroupName, MessageQueueAccessRights.WriteMessage, AccessControlEntryType.Allow);
```

Comment:
Thanks @mattburton for digging that up. I just verified with Windows 10 and there it also adds send permission by default for Everyone and Anonymous.